ImgShareUnitTests/AlbumTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using ImgShare.APISource.Data;
+using ImgShare.APISource.Data.ImgurResponseModels;
+using ImgShare.APISource.UnitTests.TestBase;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace ImgShare.APISource.UnitTests.Album

```

Comment:
65+% code coverage of the core library with the existing tests.  Not happy with it yet.  But better than last time (~40ish percent)